src/terraform/devcontainer-feature.json

```diff
@@ -24,8 +24,8 @@
                 "0.47.0",
                 "0.46.1"
             ],
-            "default": "0.46.1",
```

Comment:
Double checking, does verifying integrity with co-signing works with versions <= 0.46.1 ?

Reply:
Wondering if we should add a test scenario for validation.

Reply:
very good point, I shouldn't have removed the previous implementation.

Reply:
Appears like it wasn't too necessary to do this since they've been publishing the cosign signatures for a long time. Nonetheless added in the fallback 👍